src/Phinx/Console/Command/Create.php

```diff
@@ -113,8 +130,54 @@ protected function execute(InputInterface $input, OutputInterface $output)
             ));
         }
 
-        // load the migration template
-        $contents = $this->getMigrationTemplate();
+        // Get the alternative template and static class options, but only allow one of them.
+        $altTemplate = $input->getOption('template');
+        $creationClassName = $input->getOption('class');
+        if (!empty($altTemplate) && !empty($creationClassName)) {
+            throw new \InvalidArgumentException('Cannot use --template and --class');
+        }
+
+        // Verify the alternative template file's existence.
+        if (!empty($altTemplate) && !file_exists($altTemplate)) {
+            throw new \InvalidArgumentException(sprintf(
+                'The alternative template file "%s" does not exist',
+                $altTemplate
+            ));
+        }
+
+        // Verify the static class exists and that it implements the required interface.
+        if (!empty($creationClassName)) {
+            if (!class_exists($creationClassName, true)) {
```

Comment:
The default for the `$autoload` parameter is already `true`, no need to explicitly provide it.